### PR TITLE
Enable dual-coverage reporting!

### PR DESCRIPTION
**Pull request by staticfloat**

### CODE REVIEW

File: master/coverage.py [hunk 1]
```diff
@@ -44,9 +47,14 @@
         "author_email" => "%(prop:authoremail)s",
     )
 )
-println("git_info: ")
-println(git_info)
-Coveralls.submit_token(r, git_info)
+
+# Submit to Coveralls
+ENV['REPO_TOKEN'] = ENV['COVERALLS_REPO_TOKEN']
```
Comment (tkelman) [inline master/coverage.py]: this'll need to be double quotes, yeah?

Comment (staticfloat) [inline master/coverage.py]: Good catch!
-E

**Comment by kshyatt** [comment]
:+1:

**Comment by staticfloat** [comment]
A'ight, let's try it!

**Comment by tkelman** [comment]
did I ask this somewhere else? coverage builder hasn't run in a few days http://buildbot.e.ip.saba.us:8010/builders/coverage_ubuntu14.04-x64

File: master/coverage.py [hunk 2]
```diff
@@ -123,15 +135,19 @@
         command=[Interpolate("julia-%(prop:shortcommit)s/bin/julia"), "-e", analyze_cov_cmd]
     ),
     ShellCommand(
-        name="Move coverage.jld -> coverage_noninlined.jld",
-        command=["mv", Interpolate("julia-%(prop:shortcommit)s/share/julia/coverage.jld"), Interpolate("julia-%(prop:shortcommit)s/share/julia/coverage_noninlined.jld")]
+        name="Move coverage_coveralls.jld -> coverage_coveralls_noninlined.jld",
+        command=["mv", Interpolate("julia-%(prop:shortcommit)s/share/julia/coverage_coveralls.jld"), Interpolate("julia-%(prop:shortcommit)s/share/julia/coverage_coveralls_noninlined.jld")]
```
Comment (tkelman) [inline master/coverage.py]: Is this supposed to just be `coveralls.jld`? Similarly with `codecov.jld` below?

Comment (staticfloat) [inline master/coverage.py]: Whoops, yeah, I missed these.

**Pull request merged**